app_dart/lib/src/service/scheduler.dart

```diff
@@ -124,7 +124,15 @@ class Scheduler {
       return;
     }
 
-    final CiYaml ciYaml = await getCiYaml(commit);
+    late CiYaml ciYaml;
```

Comment:
We don't need to pass the tot config in this case (only for the presubmit targets)